src/services/blocks/BlocksTraceService.ts

```diff
@@ -0,0 +1,102 @@
+import { BlockHash } from '@polkadot/types/interfaces';
+import { InternalServerError } from 'http-errors';
+
+import {
+	BlocksTrace,
+	BlocksTraceOperations,
+} from '../../types/responses/BlocksTrace';
+import { AbstractService } from '../AbstractService';
+import { BlockTraceResponse, isBlockTrace, isTraceError, Trace } from './trace';
+
+const DEFAULT_TARGETS = 'pallet,frame,state';
```

Comment:
This seems extremely odd as an input, is this how `this.api.rpc.state.traceBlock()` expects the input?

Reply:
Yup, check out the RPC docs: https://github.com/paritytech/substrate/pull/7780/files?file-filters%5B%5D=.lock&file-filters%5B%5D=.md&file-filters%5B%5D=.rs#diff-a470f7227faf378aa4b8cf63f5b3d35018da37803f6bf0456327e4f6d2e15250R140-R244

Reply:
I don't want to link to the `env_logger` docs since we do custom filtering (prefix based), but I will link the RPC docs.

Reply:
yeah or just write a comment like:

> we use the same default tracing targets as substrate, for further tracing targets [link to substrate or something]